src/Module.php

```diff
@@ -98,6 +108,11 @@ class Module extends \yii\base\Module implements BootstrapInterface
      * @since 2.0.14
      */
     public $disableIpRestrictionWarning = false;
+    /**
+     * @var bool whether to disable access callback restriction warning triggered by checkAccess function
+     * @since 2.1.0
+     */
+    public $disableCallbackRestrictionWarning = false;
```

Comment:
Why do we need it?

Reply:
not necessary, just did it beacause we already have one for [IP restriction](https://github.com/yiisoft/yii2-debug/blob/4cc02c14f0e204e5895a124df59da07d0b3a48dc/src/Module.php#L110)

Reply:
Ah. Right. That's consistent.